.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
`*` -> `*.{html,js,json}`

Reply:
So that `utf‑8` encoding and `LF` line endings are enforced everywhere.